game/src/engine/render/Mesh.h

```diff
@@ -2,10 +2,12 @@
 
 #include <vector>
 
+#include "engine/core/gfx/Texture.h"
 #include "engine/core/gfx/Vertex.h"
 
 struct Mesh
 {
     std::vector<Vertex> vertices;
     std::vector<uint32_t> indices;
+    std::vector<Texture> textures;
```

Comment:
On the rendering side, I'm gonna implement textures as a sort of "material" to make it be separate from the Mesh, so we probably won't need the Mesh to keep any info on textures